prowler/providers/aws/services/eks/eks_cluster_ensure_version_is_supported/eks_cluster_ensure_version_is_supported.py

```diff
@@ -0,0 +1,39 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.eks.eks_client import eks_client
+
+class eks_cluster_ensure_version_is_supported(Check):
+    def execute(self) -> Check_Report_AWS:
+        findings = []
+
+        for cluster in eks_client.clusters:
+            report = Check_Report_AWS(self.metadata())
+            report.status = "PASS"
+            report.region = cluster.region
+            report.resource_id = cluster.name
+            report.resource_arn = cluster.arn
+            report.resource_tags = cluster.tags
+            report.status_extended = f"EKS cluster {cluster.name} version is supported."
+
+            user_version_num = cluster.version.split(".")
+
+            eks_latest_version = eks_client.audit_config.get(
```

Comment:
I think that could be easier to use a list with the supported versions by EKS and find the current cluster version in the list. Here is a [reference with the supported version](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)